app/src/main/java/com/example/taskplanner/presentation/custom_view/CustomTextInputLayout.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.taskplanner.presentation.custom_view
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.inputmethod.EditorInfo
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.example.taskplanner.R
+import com.google.android.material.textfield.TextInputEditText
+import com.google.android.material.textfield.TextInputLayout
+
+class CustomTextInputLayout @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet,
+    defAttrs: Int = 0
+) : ConstraintLayout(context, attrs, defAttrs) {
+    var textInputLayout: TextInputLayout
+    var editText: TextInputEditText
+    private var attributes: TypedArray
+    private var inputType: Int
+    private var maxLines: Int
+    private var hintText: String
+
+    init {
+        inflate(context, R.layout.custom_text_input_layout, this)
+        attributes = context.theme.obtainStyledAttributes(
+            attrs,
+            R.styleable.CustomTextInputLayout,
+            defAttrs,
+            0
+        )
+        textInputLayout = findViewById(R.id.rootLayout)
+        editText = findViewById(R.id.rootEditText)
```

Comment:
binding-ის ცვლადი გამოაცხადე და გაუტოლე ItemBinding.inflate(
            LayoutInflater.from(
                context
            ), this, true
        )
და აღარ დაგჭირდება findViewById